net-libs/libslirp/libslirp-4.7.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A TCP-IP emulator used to provide virtual networking services"
+DESCRIPTION="libslirp is a general purpose TCP-IP emulator used by virtual machine hypervisors, \
+containers, and emulation software to provide virtual networking services."
+HOMEPAGE="https://gitlab.freedesktop.org/slirp/libslirp/"
+COPYRIGHT="1995, 1996 Danny Gasparovski"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE-/archive/v$portVersion/libslirp-v$portVersion.tar.gz"
+CHECKSUM_SHA256="9398f0ec5a581d4e1cd6856b88ae83927e458d643788c3391a39e61b75db3d3b"
+SOURCE_DIR="libslirp-v$portVersion"
+PATCHES="libslirp-$portVersion.patchset"
+
+ARCHITECTURES="all ?x86_gcc2"
```

Comment:
You can probably just disable x86_gcc2 as a primary and remove it from SECONDARY_ARCHITECTURES.  The port will need patching to make it c++98 compatible.